README.md

```diff
@@ -26,6 +26,22 @@
   <sub>Built with 🦀🕸 by <a href="https://rustwasm.github.io/">The Rust and WebAssembly Working Group</a></sub>
 </div>
 
+## Install `wasm-bindgen-cli`
+
+If you have [`cargo-binstall`](https://github.com/cargo-bins/cargo-binstall) installed,
+then you can install the pre-built artifacts by running:
+
+```
+cargo binstall wasm-bindgen-cli
+```
+
+Otherwise, you can either download it from the
+[release](https://github.com/rustwasm/wasm-bindgen/releases) page or run:
+
+```
+cargo install wasm-bindgen-cli
+```
```

Comment:
I don't feel entirely comfortable promoting a third party tool I know nothing about.
Could you change the order like this:
1. `cargo install wasm-bindgen-cli`
2. GitHub Release Artifacts
3. `cargo-binstall`

Thank you by the way, improving the documentation is sorely needed.

Reply:
Thanks, I've changed the order as requested.